server/pkg/builtin/manifest_ja.yml

```diff
@@ -148,9 +145,6 @@ extensions:
           camera:
             title: カメラ初期位置
             description: ページロード後最初に表示するカメラの位置を設定します。
-          sceneMode:
-            title: シーンモード
-            description: 地球儀の表示を2D、2.5D、3Dから選択します。
```

Comment:
I think this is for Classic, please keep these there, we might going to clean up these all together later.